frontend/src/screens/SignIn.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import '../App.global.scss';
+import {
+  Button,
+  Box,
+  Layout,
+  Image,
+  Heading,
+  InputWithLabel,
+  Cell,
+} from 'wix-style-react';
+import { Route } from 'react-router-dom';
+
+const SignIn = props => {
```

Comment:
Let's use destructuring: `const SignIn = ({ callback }) => { ... }`
Also, I think a more specific name for `callback` would be better